README.md

```diff
@@ -1,2 +1,213 @@
-# car_app_backend
-This is the back end for the car app.
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Car APP - Back End <a name="about-project"></a>
+
+** Car APP - Back End ** Is a Ruby Back-End project for users to rent cars, and Admins to create and delete cars.  Users will have the chance to view the history of cars reserved.
+
+** The app front end can be found [here](https://github.com/gjuliao/car_app_frontend)
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/documentation/">Ruby on Rails</a></li>
+  </ul>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Users can view Cars signup, login and reserve cars.**
+- **Admin can login, add and delete cars.**
+- **Users can view history of cars**
+- **Users can view characteristics of cars (price, electric, brand and much more).**
+- **Users can view which cars are available for rent and which aren't**
+- **Full stack app**
+- **Styling uses CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running
+- [PostgreSQL](https://www.postgresql.org/) installed
+- [RubyonRails](https://rubyonrails.org/) installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``` bash
+git clone git@github.com:dgcuenca/car_app_backend.git
+```
+and then move to the folder: 
+
+```bash
+cd car_app_backend
+```
+
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install dependancies with:
+
+```bash
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Create a master.key file in the config folder and paste  the key that was provided
+```
+code ./config/master.key
+```
```

Comment:
- [ ] In order for all your users to be able to run the project, we need instructions about how to create our own master.key locally.  You don't need to share the key, just provide instructions about what needs to be done in order to create it.  These are the instructions:

```
1. Delete config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Kindly, use these instructions on your readme, you won't need to share the key.

That way, all your users will be able to setup and run the project.  These users may include people outside Microverse, like potential employers eager to test your project.